apps/src/templates/DropdownButton.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component, PropTypes } from 'react';
+import ReactDOM from 'react-dom';
+import Radium from 'radium';
+import color from "@cdo/apps/util/color";
+import Button from '@cdo/apps/templates/Button';
+
+const styles = {
+  main: {
+    display: 'inline-block',
+  },
+  icon: {
+    fontSize: 24,
+    // we want our icon text to be a different size than our button text, which
+    // requires we manually offset to get it centered properly
+    position: 'relative',
+    top: 3
+  },
+  dropdown: {
+    border: `1px solid ${color.charcoal}`,
+    position: 'absolute',
+    // without this, this will be below some content
+    zIndex: 1,
+  },
+  section: {
```

Comment:
It had been (this was copied over), but the style name should probably be updated now. good catch.